cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -201,6 +201,13 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
       queryText = "CREATE CONSTRAINT ON (book:Book) ASSERT book.title IS NOT NULL",
       assertions = _ => hasNodeConstraint("Book", "title")
     )
+    testQuery(
+      title = "Create a node property existence constraint using deprecated syntax",
```

Comment:
I don't think these can have the same title/name as the other one if we want to show both...